src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,56 @@
+import { Component } from 'react';
+
+type State = {
+  today: Date,
+};
+
+type Props = {
+  name: string,
+};
+
+export class Clock extends Component<Props, State> {
+  state: Readonly<State> = {
```

Comment:
I'm not sure we need extra type of 'state', but I could be wrong

Reply:
you are changing state on 22 line, I think it is not readonly